tools/cli/commands/rsync.js

```diff
@@ -0,0 +1,199 @@
+/**
+ * External dependencies
+ */
+import fs from 'fs';
+import process from 'process';
+import chalk from 'chalk';
+import execa from 'execa';
+import inquirer from 'inquirer';
+import watch from 'node-watch';
+import tmp from 'tmp';
+/**
+ * Internal dependencies
+ */
+import { projectDir } from '../helpers/install.js';
+import { listProjectFiles } from '../helpers/list-project-files.js';
+import { allProjectsByType, dirs } from '../helpers/projectHelpers.js';
+import { runCommand } from '../helpers/runCommand.js';
+
+/**
+ * Entry point for the CLI.
+ *
+ * @param {object} argv - The argv for the command line.
+ */
+export async function rsyncInit( argv ) {
+	argv = await promptForPlugin( argv );
+	argv = await promptForDest( argv );
+	const sourcePluginPath = projectDir( `plugins/${ argv.plugin }` );
```

Comment:
rsync is a bit picky when it comes to use or non-use of trailing slashes. To avoid having to repeat the plugin name on every line of the filter, we need to do like this here
```suggestion
	const sourcePluginPath = projectDir( `plugins/${ argv.plugin }` ) + '/';
	argv.dest = path.join( argv.dest, argv.plugin + '/' );
```

Reply:
> rsync is a bit picky
Indeed. Great suggestion